core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -432,7 +434,7 @@ class KafkaController(val config: KafkaConfig,
       val newVersion = createFeatureZNode(
         FeatureZNode(config.interBrokerProtocolVersion,
           FeatureZNodeStatus.Enabled,
-          brokerFeatures.defaultFinalizedFeatures
+          brokerFeatures.defaultFinalizedFeatures.map { case (k, v) => (k, v.shortValue()) }
```

Comment:
`brokerFeatures.defaultFinalizedFeatures.asScala.map { case (k, v) => (k, v.shortValue()) }`
In this way we could avoid using deprecated library and remove this import:
`import scala.collection.convert.ImplicitConversions.map AsScala`